app/cypress/fixtures/secureHrefTests.js

```diff
@@ -0,0 +1,127 @@
+import minimalDoc from './documentTests/shared/minimalDoc.js'
+
+export default [
+  // Valid tests
+  {
+    title: '# href (valid)',
+    valid: true,
+    content: {
+      document: {
+        ...minimalDoc.document,
+        acknowledgments: [{ urls: ['#test.org'] }],
+      },
+    },
+  },
+  {
+    title: 'http href (valid)',
+    valid: true,
+    content: {
+      document: {
+        ...minimalDoc.document,
+        acknowledgments: [{ urls: ['http://test.org'] }],
+      },
+    },
+  },
+  {
+    title: 'https href (valid)',
+    valid: true,
+    content: {
+      document: {
+        ...minimalDoc.document,
+        acknowledgments: [{ urls: ['https://test.org'] }],
+      },
+    },
+  },
+  {
+    title: 'mailto href (valid)',
+    valid: true,
+    content: {
+      document: {
+        ...minimalDoc.document,
+        acknowledgments: [{ urls: ['mailto://user@test.org'] }],
+      },
+    },
+  },
+  {
+    title: 'tel href (valid)',
+    valid: true,
+    content: {
+      document: {
+        ...minimalDoc.document,
+        acknowledgments: [{ urls: ['tel://+4934587913248test.org'] }],
+      },
+    },
+  },
+  {
+    title: 'ftp href (valid)',
+    valid: true,
+    content: {
+      document: {
+        ...minimalDoc.document,
+        acknowledgments: [{ urls: ['ftp://test.org'] }],
+      },
+    },
+  },
+  {
+    title: 'data href (MIME-type png) (valid)',
+    valid: true,
+    content: {
+      document: {
+        ...minimalDoc.document,
+        acknowledgments: [{ urls: ['data:image/png;base64,iVBORwtest.org0KGgoAAAANSUhEUgAAA'] }],
```

Comment:
Hmm - that doesn't look like a valid base64 to me. But the idea is good  - we should cover both cases (invalid and valid base64).